corsixth-appdata.patch

```diff
@@ -0,0 +1,55 @@
+From baec9533810026e55554ce7a6dc8e1817173ca88 Mon Sep 17 00:00:00 2001
+From: Toby <mrtobylane@gmail.com>
+Date: Wed, 26 Aug 2020 00:43:31 +0100
+Subject: [PATCH] Update appdata.xml (#1693)
+
+---
+ ...ppdata.xml => com.corsixth.corsixth.appdata.xml} | 13 +++++++------
+ 1 file changed, 7 insertions(+), 6 deletions(-)
+ rename CorsixTH/{com.corsixth.CorsixTH.appdata.xml => com.corsixth.corsixth.appdata.xml} (88%)
+
+diff --git a/CorsixTH/com.corsixth.CorsixTH.appdata.xml b/CorsixTH/com.corsixth.corsixth.appdata.xml
+similarity index 88%
+rename from CorsixTH/com.corsixth.CorsixTH.appdata.xml
+rename to CorsixTH/com.corsixth.corsixth.appdata.xml
+index 68819ccd94..b252d5bc78 100644
+--- a/CorsixTH/com.corsixth.CorsixTH.appdata.xml
++++ b/CorsixTH/com.corsixth.corsixth.appdata.xml
+@@ -1,6 +1,6 @@
+ <?xml version="1.0" encoding="UTF-8"?>
+ <component type="desktop">
+-  <id>com.corsixth.CorsixTH</id>
++  <id>com.corsixth.corsixth</id>
+   <!-- Copyright 2019 Artem Polishchuk <ego.cordatus@gmail.com> -->
+   <metadata_license>CC0</metadata_license>
+   <project_license>BSD-3-Clause AND MIT</project_license>
+@@ -24,14 +24,14 @@
+     <binary>corsix-th</binary>
+   </provides>
+   <releases>
+-    <release version="0.63" date="2019-05-24"></release>
++    <release version="0.64" date="2020-06-16"></release>
```

Comment:
Did you submit these upstream?

Reply:
The two CorsixTH patches are actually upstream commits that I cherry-picked:
- https://github.com/CorsixTH/CorsixTH/commit/baec9533810026e55554ce7a6dc8e1817173ca88
- https://github.com/CorsixTH/CorsixTH/commit/8962dfc7880d08ee77c2f81dacdb30c9114f2204

They will be dropped with the next upstream release.

Reply:
Excellent.

Reply:
Did you also need https://github.com/CorsixTH/CorsixTH/commit/b595595439726f35b36004cfd6f41b3f61a3a009